tests/Microsoft.Identity.Test.UIAutomation/XamarinDroidMSALTests.cs

```diff
@@ -109,15 +109,24 @@ public void AcquireTokenSilentTest()
             _msalMobileTestHelper.AcquireTokenSilentTestHelper(xamarinController, LabUserHelper.GetDefaultUser());
         }
 
+        /// <summary>
+        /// Runs through the standard acquire token flow with an MSA user, using the default app configured with UiBehavior = Login
+        /// </summary>
+        [Test]
+        public void AcquireTokenMsaUserTest()
```

Comment:
>AcquireTokenMsaUserTest [](start = 20, length = 23)

isn't it possible to simply run all the test cases (except B2C) using both an AAD account and an MSA account? ... Suggestion, don't add a test specifically for this, but instead ensure all tests runs with both user types?